src/model/message.model.ts

```diff
@@ -21,13 +22,13 @@ export function isHide(message: any): message is Hide {
 }
 
 
-export interface ManualConnection {
-    manual_connection: boolean
+export interface ManualConnectionScreen {
+    manual_connection_screen: boolean
```

Comment:
why snake case suddenly?  you seem to be using camel case everywhere else

Reply:
from the start project, I use snake case for the webviewApi In the request to display the html screens
Would you like to change it all to camel case?

Reply:
> why snake case suddenly? you seem to be using camel case everywhere else

but I now changed all the 'webviewApi' attributes to camel case to make all project style the same and also changed some variables' names that may be confusing.